demystifying_geospatial_data/demystifying_geospatial_data.md

```diff
@@ -0,0 +1,221 @@
+
+While GPS has greatly improved the quality of geospatial data, it is still not perfect and can have the same sorts of missing and nonsensical fields as any data set. For example, [Philadelphia records of police stops](github_link) record some small portion of stops as occurring in Cape Canaveral, Florida, approximately a thousand miles away.
+
+The abundance of geospatial data created by modern systems also creates new problems. While it might seem like more data is always better, even the most high powered computer can only handle so much data at a time.
+
+A single weather forecast, for example, incorporates a huge amount of real-time geospatial data, most of it recorded by satellites in low orbit around the Earth.
+
+Of course there is far more location data than that collected by satellites.
+
+
+### Privacy
+
+As with all data, privacy and ownership of people's geospatial data is an important consideration. You might be personally aware of how much location data your devices, like your cell phone, are recording.
+
+**Patient and participant privacy**
+
+The maps of cholera deaths from 1854 might not contain the names of the individuals who died, but they do contain enough information to reconstruct a pretty good picture of which individuals are represented simply by looking at who lives, or lived at each address. As a researcher using geospatial data, you have additional responsibility to keep data private.
+
+One way to do that is to only publishing location data in aggregate. For example you might take patient addresses and only record which postal code or other larger geographic region they are in.
+
+Another method to preserve privacy is called **jittering**. Jittering means adding a small amount of random noise to the real location data so that the jittered data are close to, but not exactly in, their true locations. With this method you can publish maps and data that look like they disclose individual addresses, but don't actually show where individuals truly live.
+
+Here is a map from a [CHOP Policy Lab article](https://www.sciencedirect.com/science/article/pii/S1876285922001814?via%3Dihub) on how neighborhood social context might increase or decrease the stress levels of mothers of young children.
+
+![Map of Philadelphia color coded by the rate of violent crime. Addresses of study participant are marked on the map with yellow dots.](media/phillystressmap.jpg)
+
+While the **Participant Address** locations might appear at first glance to give away exactly where each participant lives, the image caption is reassuring: "Participant locations were randomly jittered and enlarged to ensure confidentiality."
+
+<div class = "warning">
+
+An internal review board, funding agency, or local law may lay out how you are required to anonymize and protect location data in your research. Be sure to follow all required steps to protect personal data when conduction research with geospatial data!
+
+</div>
+
+### Expense
```

Comment:
This is very brief -- can you enrich, or roll into an earlier page, so it's not so much whitespace on the page?

Reply:
folded in as a subsection of another page